src/components/BulkEnrollmentPage/data/reducer.js

```diff
@@ -0,0 +1,14 @@
+import { SET_SELECTED_ROWS, DELETE_ROW } from './actions';
+
+const selectedRowsReducer = (state = [], action) => {
```

Comment:
STUDY/LOOK INTO: obviously let's not worry about it for this PR but if we use action and reducers code, we may want to look into using reduxjs/toolkit to reduce some of our actions/reducers boilerplate. e.g., `createSlice` takes care of reducing an amazing amount of code (esp more apparent where there are lots of actions)
https://redux-toolkit.js.org/api/createSlice while keeping all actions and reducers usage intact.